x-pack/legacy/plugins/epm/server/datasources/create.ts

```diff
@@ -97,10 +109,30 @@ async function getDatasource(options: {
   return datasource?.attributes;
 }
 
-function createFakeDatasource(pkg: RegistryPackage, assets: Asset[] = []): Datasource {
+function createFakeDatasource(
```

Comment:
Now that this is 4 arguments, I'd like to convert it to an object. The names you have here are good, IMO (plus previous comment about `datasets`). `{ pkg, datasourceName, datasets, assets }`